system/system_monitor/src/net_monitor/net_monitor.cpp

```diff
@@ -124,7 +124,7 @@ void NetMonitor::checkUsage(diagnostic_updater::DiagnosticStatusWrapper & stat)
 
       ++index;
       close(fd);
-      entried.push_back(ifa->ifa_name);
+      appended.push_back(ifa->ifa_name);
```

Comment:
I think `interface_names` is better.

Reply:
Changed in f06568c1dc43c07e8b54747d0c09b02bf0a088c5